plugins/out_loki/loki.h

```diff
@@ -91,6 +92,7 @@ struct flb_loki {
     struct flb_record_accessor *ra_k8s;      /* kubernetes record accessor */
     struct mk_list labels_list;              /* list of flb_loki_kv nodes */
     struct mk_list structured_metadata_list; /* list of flb_loki_kv nodes */
+    struct mk_list structured_metadata_map_keys_list; /* list of flb_loki_kv nodes */
```

Comment:
Do the comment start positions need to be aligned, or is it okay for this one to stick out further?

Reply:
I'd say line them up if you can

Reply:
seems reasonable to me